game/game.py

```diff
@@ -47,9 +47,14 @@
 class WumpusGame():
 
     def __init__(self) -> None:
+
+        # --------------------------------
         self._board = [
             [Cell(i, j) for j in range(LARGE)] for i in range(LARGE)
         ]
+
+        # --------------------------------
+
```

Comment:
nit: this kind of comments aren't used in PRODUCTION environments (at Eventbrite)
```suggestion
```